src/components/Driver/DriverPage.js

```diff
@@ -32,7 +32,8 @@ const DriverPage = () => {
     axios.post(`${baseUrl}/coordinates/confirm`, {
       id: userId,
     })
```

Comment:
not a part of testing, consider separating PR's in the future. Just a suggestion tho, everything else seems okay :)